Ensure that CONFIG_SANDBOX is set when running "make env", avoiding a
failure to build caused by config_distro_bootcmd.h following the wrong
codepath...

Index: u-boot/include/configs/sandbox.h
===================================================================
--- u-boot.orig/include/configs/sandbox.h
+++ u-boot/include/configs/sandbox.h
@@ -66,6 +66,9 @@
 #ifdef __ASSEMBLY__
 #define BOOTENV
 #else
+#ifndef CONFIG_SANDBOX
+#define CONFIG_SANDBOX 1
+#endif
 #include <config_distro_bootcmd.h>
 #endif
 
